### PR TITLE
community[patch]: Add docstring for Links

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

**Comment by cbornet** [comment]
Hello, could this get some eyes ?
Only docstring so should be an easy review.

**Comment by cbornet** [comment]
Rebased to use langchain_community.

**Review by eyurtsev — APPROVED** [review]

File: libs/community/langchain_community/graph_vectorstores/links.py [hunk 1]
```diff
@@ -9,10 +9,113 @@
...
+    +----------+----------+---------------------------------+
+
+    Directed links make it possible to describe relationships such as term
+    references / definitions: term definitions are generally relevant to any documents
+    that use the term, but the full set of documents using a term generally aren't
+    relevant to the term's definition.
+
+    .. seealso::
+
+        - :mod:`How to use a graph vector store <langchain_community.graph_vectorstores>`
```
Comment (eyurtsev) [inline libs/community/langchain_community/graph_vectorstores/links.py:56]: cool -- curious to see how well all of these will render on the APIReference page!

Comment (cbornet) [reply libs/community/langchain_community/graph_vectorstores/links.py:56]: You can see now: https://python.langchain.com/api_reference/community/graph_vectorstores/langchain_community.graph_vectorstores.links.Link.html 😉

**Pull request merged**